### PR TITLE
Update pl.js

### PR DESCRIPTION
**Pull request by xhg4**
Note to the reviewer: I have no idea how to translate line 523, if you have an idea for a translation of that line, please let me know by simply making a comment under this PR

### CODE REVIEW

**Comment by Isori1** [comment]
Line 523, translated to 'Wybierz Przykrywę (przytrzymaj)'

File: pl.js [hunk 1]
```diff
@@ -513,14 +513,14 @@ windows.controls.specFree Przełącz tryb wolnej kamery
...
-windows.controls.specFocus Toggle Focus Player
-windows.controls.hidePlayers Hide Players
+windows.controls.specFocus Zmień gracza, z którego perspektywy oglądasz rozgrywkę
+windows.controls.hidePlayers Ukryj graczy
 windows.controls.streak0 Seria 1
 windows.controls.streak1 Seria 2
 windows.controls.streak2 Seria 3
 windows.controls.streak3 Seria 4
 windows.controls.streak4 Seria 5
-windows.controls.prop Select Prop (Hold)
```
Comment (Isori1) [inline pl.js:523]: Line 523, 'Wybierz Przykrywę (przytrzymaj)'

**Review by Isori1 — COMMENTED** [review]
Wybierz Przykrywę (przytrzymaj)

**Comment by xhg4** [comment]
> Wybierz Przykrywę (przytrzymaj)

Already changed that in the latest commit, you've probably looked at the initial commit in this pull request

**Comment by Isori1** [comment]
This PR has been Reviewed and approved.

**Comment by xhg4** [comment]
@mty22 PR ready to be merged

**Comment by mty22** [comment]
kr queued

**Pull request merged**